### PR TITLE
Small fixes for iTeamTalk

### PR DESCRIPTION
**Pull request by Oreonan**
- Add an accessibility value to say if tx button is on or off
- Return who has kicked
- Change title for private messages to return nickname
Unfortunately I can't test changes for iOS at moment. @bear101 please let me know if something should be change. If it's OK for you, could you update translations, after that a new beta will be appreciated.

### CODE REVIEW

**Comment by ChrisDuffley** [comment]
If someone is willing to post a build in TestFlight, I'd be more than happy to test with iOS. In fact, I'm using it right now to reply to this email.

Thanks!

Christopher Duffley
Host of Mission Possible Podcast: http://www.missionpossiblepodcast.com
Cell: +1 (603) 851-6971

**Comment by Oreonan** [comment]
You can already test latest beta in TestFlight:

https://testflight.apple.com/join/ZPILmexZ

File: Client/iTeamTalk/iTeamTalk/ChannelListViewController.swift [hunk 1]
```diff
@@ -674,7 +674,7 @@ class ChannelListViewController :
 
         let src_vc = segue.source as! UserDetailViewController
         
-        let vc = self.storyboard?.instantiateViewController(withIdentifier: "Text Message") as! TextMessageViewController
+        let vc = self.storyboard?.instantiateViewController(withIdentifier: format("Message - %@", getDisplayName(src_vc.userid)) as! TextMessageViewController
```
Comment (bear101) [inline Client/iTeamTalk/iTeamTalk/ChannelListViewController.swift:677]: "Text Message" is the ID of a View Controller in the Story Board.
Can you remove this last commit? The commits before looks ok.

File: Client/iTeamTalk/iTeamTalk/ChannelListViewController.swift [hunk 2]
```diff
@@ -1024,7 +1024,7 @@ class ChannelListViewController :
                 }
                 
                 if settings.object(forKey: PREF_DISPLAY_POPUPTXTMSG) == nil || settings.bool(forKey: PREF_DISPLAY_POPUPTXTMSG) {
-                    let vc = self.storyboard?.instantiateViewController(withIdentifier: "Text Message") as! TextMessageViewController
+                    let vc = self.storyboard?.instantiateViewController(withIdentifier: format("Message - %@", getDisplayName(txtmsg.nFromUserID))) as! TextMessageViewController
```
Comment (bear101) [inline Client/iTeamTalk/iTeamTalk/ChannelListViewController.swift:1027]: Same here

**Comment by Oreonan** [comment]
Yes, I can :), but in this case what is the best practice to change 
title of MP window to display nickname of user, if it's possible?

**Pull request merged**